pkg/skaffold/deploy/helm/helm.go

```diff
@@ -398,7 +398,11 @@ func (h *Deployer) Dependencies() ([]string, error) {
 			return true, nil
 		}
 
-		if err := walk.From(release.ChartPath).When(isDep).AppendPaths(&deps); err != nil {
+		expanded_path, e := util.ExpandEnvTemplateOrFail(release.ChartPath, nil)
```

Comment:
Please fix this lint issue, `expanded_path` should be `expandedPath`

Reply:
appologies. I did not notice the lint check before.
Fixed.